docs/development.rst

```diff
@@ -448,6 +448,145 @@ finally force-push to the remote topic branch on your fork::
+        _chromosomes.append(stdpopsim.Chromosome(
+        id=name, length=int(length),
+        mutation_rate=FILL_ME,
+        recombination_rate=FILL_ME))
+
+    # A citation for the chromosome parameters. Additional citations may be needed if
+    # the mutation or recombination rates come from other sources. In that case create
+    # additional citations with the appropriate reasons specified (see API documentation
+    # for stdpopsim.citations)
+
+    _assembly_citation = stdpopsim.Citation(
+        doi="FILL ME",
+        year="FILL ME",
+        author="Author et al.",
+        reasons={stdpopsim.CiteReason.ASSEMBLY})
+
+    # Create a genome object
+
+    _genome = stdpopsim.Genome(
+        chromosomes=_chromosomes,
+        assembly_citations=[_assembly_citation])
+
+Once you have a genome object you can create a new `Species` object which contains
+species identifiers, the genome, and default generation time and population size settings
+along with the relevant citation(s). Below is an example species definition for
+Arabidopsis thaliana and a final line of code that registers the species in the catalog.
+
+.. code-block:: python
+
+    _gen_time_citation = stdpopsim.Citation(
+        doi="https://doi.org/10.1890/0012-9658(2002)083[1006:GTINSO]2.0.CO;2",
+        year="2002",
+        author="Donohue",
+        reasons={stdpopsim.CiteReason.GEN_TIME})
+
+    _pop_size_citation = stdpopsim.Citation(
+            doi="https://doi.org/10.1016/j.cell.2016.05.063",
+            year="2016",
+            author="1001GenomesConsortium",
+            reasons={stdpopsim.CiteReason.POP_SIZE})
+
+    _species = stdpopsim.Species(
+        id="AraTha",
+        name="Arabidopsis thaliana",
+        common_name="A. thaliana",
+        genome=_genome,
+        generation_time=1.0,
+        generation_time_citations=[_gen_time_citation],
+        population_size=10**4,
+        population_size_citations=[_pop_size_citation]
+        )
+
+    stdpopsim.register_species(_species)
+
+Once all of this is done, go to the `catalog/__init__.py` file and add a line like the
+one below using the six-letter species identifier. Make sure to keep the comment to
+prevent linting issues.
+
+.. code-block:: python
+
+    from .catalog import PonAbe  #NOQA
+
+--------------------
+Adding a genetic map
+--------------------
+Some species have sub-chromosomal recombination maps available. If there is one for your
+species that you wish to include it is specified in a space delimited file with four
+columns: Chromosome, Position(bp), Rate(cM/Mb), and Map(cM). Each chromosome should be
+placed in a seperate file and with the chromosome id in the file name in such a way that
+it can be programatically parsed out. IMPORTANT: chromosome ids must match those provided
+in the genome definition exactly! Below is an example start to a recombination map file::
+
+    Chromosome Position(bp) Rate(cM/Mb) Map(cM)
+    chr1 32807 5.016134 0
+    chr1 488426 4.579949 0
```

Comment:
The map might still work (if msprime just uses the Rate instead of Map column, but the Map should be increasing above 0 with positive rate?

Reply:
It might be simplest to point at the [msprime docs](https://msprime.readthedocs.io/en/stable/api.html#msprime.RecombinationMap.read_hapmap) here instead of providing such detail. Stdpopsim does just call `msprime.RecombinationMap.read_hapmap()` after all.

Reply:
I'll do that

Reply:
Ok, after reading the `msprime` documentation it doesn't actually help explain much. I don't understand this bit: "The first column in this file is ignored, as are additional columns after the third (Position is assumed to be the second column, and Rate is assumed to be the third)". If that's the case why have the 4th column in the example? Also how is the first column ignored, it's essential for creating the per chromosome maps, right? I'll link to it but I think it needs some clarification.

Reply:
I think this file format already existed before msprime started using it. So the unused columns might be useful in some contexts, but they just aren't needed in msprime (or stdopsim). In the unlikely event that one file contains recombination maps for all chromosomes, then this can be split using the first column of the file. But that would need to be done outside of msprime/stdpopsim.